tests/data/pytest/record_test_result.json

```diff
@@ -1,168 +1,168 @@
-        "created_at":"2021-12-09T14:09:43.746900",
-        "data":null
+       "type": "case",
+       "testPath": [
+         {
+           "type": "file",
+           "name": "tests/data/pytest/tests/funcs3_test.py"
+         },
+         {
+           "type": "class",
+           "name": "tests.data.pytest.tests.funcs3_test"
+         },
+         {
+           "type": "testcase",
+           "name": "test_func4"
+         }
+       ],
+       "duration": 0.001,
+       "status": 1,
+       "stdout": "",
+       "stderr": "",
+       "created_at": "2021-12-09T14:09:43.746900",
+       "data": null
      },
      {
-        "type":"case",
-        "testPath":[
-           {
-              "type":"file",
-              "name":"tests/data/pytest/tests/funcs3_test.py"
-           },
-           {
-              "type":"class",
-              "name":"tests.data.pytest.tests.funcs3_test"
-           },
-           {
-              "type":"testcase",
-              "name":"test_func5"
-           }
-        ],
-        "duration":0.001,
-        "status":0,
-        "stdout":"",
-        "stderr":"",
-        "created_at":"2021-12-09T14:09:43.746900",
-        "data":null
+       "type": "case",
+       "testPath": [
+         {
+           "type": "file",
+           "name": "tests/data/pytest/tests/funcs3_test.py"
+         },
+         {
+           "type": "class",
+           "name": "tests.data.pytest.tests.funcs3_test"
+         },
+         {
+           "type": "testcase",
+           "name": "test_func5"
+         }
+       ],
+       "duration": 0.001,
+       "status": 0,
+       "stdout": "",
+       "stderr": "assert 1 == False\ndef test_func5():\n  >       assert 1 == False\n  E       assert 1 == False\n\n        tests/funcs3_test.py:6: AssertionError      ",
+       "created_at": "2021-12-09T14:09:43.746900",
+       "data": null
      },
      {
-        "type":"case",
-        "testPath":[
-           {
-              "type":"file",
-              "name":"tests/data/pytest/tests/test_funcs1.py"
-           },
-           {
-              "type":"class",
-              "name":"tests.data.pytest.tests.test_funcs1"
-           },
-           {
-              "type":"testcase",
-              "name":"test_func1"
-           }
-        ],
-        "duration":0.001,
-        "status":1,
-        "stdout":"",
-        "stderr":"",
-        "created_at":"2021-12-09T14:09:43.746900",
-        "data":null
+       "type": "case",
+       "testPath": [
+         {
+           "type": "file",
+           "name": "tests/data/pytest/tests/test_funcs1.py"
+         },
+         {
+           "type": "class",
+           "name": "tests.data.pytest.tests.test_funcs1"
+         },
+         {
+           "type": "testcase",
+           "name": "test_func1"
+         }
+       ],
+       "duration": 0.001,
+       "status": 1,
+       "stdout": "",
+       "stderr": "",
+       "created_at": "2021-12-09T14:09:43.746900",
+       "data": null
```

Comment:
Are these spaces just resulted from formatting?

Reply:
yes, except for added stderr

Reply:
ok